lib/ng-bank-parser.rb

```diff
@@ -4,8 +4,11 @@
 require "ng-bank-parser/pdf-unlocker"
 require "ng-bank-parser/parsers/gtb-excel-parser"
 require "ng-bank-parser/parsers/uba-pdf-parser"
-require "ng-bank-parser/parsers/firstbank-pdf-parser"
+require "ng-bank-parser/parsers/accessbank-pdf-parser"
```

Comment:
You removed the requirement for firstbank here. You should add access, and not remove firstbank